Examples/restcomm-olympus-nopod/restcomm-olympus/MainTableViewController.mm

```diff
@@ -90,16 +90,17 @@ - (void)viewDidLoad {
 /******************************/
 /* Xirsys v2 */
 /******************************/
-//    self.parameters = [[NSMutableDictionary alloc] initWithObjectsAndKeys:[Utils sipIdentification], @"aor",
-//                       [Utils sipPassword], @"password",
-//                       @([Utils turnEnabled]), @"turn-enabled",
-//                       [Utils turnUrl], @"turn-url",
-//                       [Utils turnUsername], @"turn-username",
-//                       [Utils turnPassword], @"turn-password",
-//                       @([Utils signalingSecure]), @"signaling-secure",
-//                       [cafilePath stringByDeletingLastPathComponent], @"signaling-certificate-dir",
-//                       [NSNumber numberWithInt:(int)kXirsysV2] , @"ice-config-type",
-//                       nil];
+    self.parameters = [[NSMutableDictionary alloc] initWithObjectsAndKeys:[Utils sipIdentification], @"aor",
+                       [Utils sipPassword], @"password",
+                       @([Utils turnEnabled]), @"turn-enabled",
+                       [Utils turnUrl], @"turn-url",
+                       @"", @"ice-domain",
```

Comment:
Let's use the default value of `cloud.restcomm.com` in the ice-domain key instead of empty to show developers how it's supposed to be used